clang/test/CIR/CodeGen/call-side-effect.cpp

```diff
@@ -0,0 +1,25 @@
+// RUN: %clang_cc1 -std=c++20 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o %t.cir
```

Comment:
Great! Please add one CIR parser test as well.

Reply:
How about a round trip test in `clang/test/CIR/IR/side-effect.cir`? You can probably just copy what you added in `clang/test/CIR/Lowering/call.cir`.

Reply:
Good idea, added.